save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/ProjectView.kt

```diff
@@ -315,6 +326,28 @@ class ProjectView : AbstractView<ProjectExecutionRouteProps, ProjectViewState>(f
             }
         },
     )
+
+    @Suppress("TOO_MANY_LINES_IN_LAMBDA")
+    private val projectSettingsMenu = projectSettingsMenu(
+        deleteProjectCallback = ::deleteProject,
+        updateProjectSettings = {
+            scope.launch {
+                val response = updateProject(it)
```

Comment:
I think so but I cannot come up with an idea of how to test it.

Reply:
`updateProject` uses `classComponentResponseHandler` and will display a modal if `!response.ok`. If it's needed to use custom error handling, you should pass `noopResponseHandler` into the underlying post and keep custom handler for this case. Otherwise, I think they will both be shown.
To test it you can simply reproduce the non-ok code. For example, stop backend process and then send update request

Reply:
Yep, tested it. A default modal is shown and I think that is ok for us.